opal/mca/accelerator/accelerator.h

```diff
@@ -323,9 +323,11 @@ typedef opal_accelerator_base_module_t * (*mca_accelerator_base_component_init_f
  * during the lifetime of the component, including outstanding
  * modules.
  *
+ * @param  module     Allow the component to perform cleanup in module
```

Comment:
The style is different from the rest of the comments. I'm also not clear if this means the component is supposed to free the module or not

Reply:
If the component performed allocation in the module class, we give it the option to clean that allocation up.

Reply:
Updated comment description to be more clear, and added [in] to match style